### PR TITLE
Made the service disable command more complete.

### PR DESCRIPTION
**Pull request by matejak**
As discussed in #2686, this PR changes the remediation, so the discussed use cases work without modifying OVAL.

I have also removed some excessive negations that were present.

### CODE REVIEW

File: shared/bash_remediation_functions/service_command.sh [hunk 1]
```diff
@@ -47,20 +47,24 @@ else
...
+if [ "x$chkconfig_util" != x ] ; then
   $service_util $service $service_operation
   $chkconfig_util --level 0123456 $service $chkconfig_state
 else
   $service_util $service_operation $service
   $service_util $service_state $service
+  # The service may not be running because it has been started and failed,
+  # so let's reset the state so OVAL checks pass.
+  # Service should be 'inactive', not 'failed' after reboot though.
+  $service_util reset-failed $service
```
Comment (mpreisler) [inline shared/bash_remediation_functions/service_command.sh]: does this work with the "service" command? this looks like something that is systemctl only.

Comment (matejak) [reply shared/bash_remediation_functions/service_command.sh]: The shell is not clear, at first sight, but I believe that the branch of the if statement is the exclusive `systemctl` branch.

Comment (redhatrises) [reply shared/bash_remediation_functions/service_command.sh]: Haven't tested but does OVAL really report pass/fail on this?

Comment (matejak) [reply shared/bash_remediation_functions/service_command.sh]: @redhatrises The PR #2686 exposes that the OVAL check on whether a service is disabled will fail is the service is indeed disabled, but in a `failed` state. Or you meant something else?

Comment (redhatrises) [reply shared/bash_remediation_functions/service_command.sh]: @matejak no didn't mean something else... I think I need more coffee as I am confusing the difference between the two PRs.

**Comment by mpreisler** [comment]
@openscap-ci test this please

**Comment by mpreisler** [comment]
ACK

**Pull request merged**